### ISSUE TITLE
How do you think about big data can predict future?

### ISSUE THREAD

**Issue by Herny88** [suniumo92/joie3#3]
Do you have some ideas about this conception?

**Comment by suniumo92** [comment]
Yes, I think there are so many companies using big data to predict different trends and the prediction also can help them do some important decisions.

**Comment by Herny88** [comment]
From my point of view, I think the big data can predict some particular future events, such as the weather, traffic flow, diseases, country's economy, finance trend and so on. However, it is very hard to predict the people's specific behaviour. Although lots of internet companies  are doing this at present, the prediction result is not always accurate. Probably, in the nearly future, with the development of machine learning technology, hardware performance, data collecting, Human would overcome this bottleneck.

**Comment by HEYJOIE** [comment]
Data analysis, based on the collation of data mining, according to the set of all the indexes data and orientation data compiled data analysis, you need big data analysis represents the behavior of the majority of users but can't say 100%, everything is uniqueness and contingencies, so the predict the future is probability. The probability is higher,the possible will be higher. The stock market has forecasting software for many years, but it has not been accurately predicted.
